com.mikrotik.WinBox.yml

```diff
@@ -0,0 +1,61 @@
+id: com.mikrotik.WinBox
+runtime: org.freedesktop.Platform
+runtime-version: "24.08"
+sdk: org.freedesktop.Sdk
+command: winbox
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --device=dri
+  - --share=network
+
+modules:
+  - name: WinBox
+    buildsystem: simple
+    build-commands:
+    - install -Dm755 apply_extra /app/bin/apply_extra
+    - install -Dm755 winbox /app/bin/winbox
+    - install -Dm644 winbox.desktop /app/share/applications/${FLATPAK_ID}.desktop
+    - desktop-file-edit --set-icon=${FLATPAK_ID} /app/share/applications/${FLATPAK_ID}.desktop
+    - install -Dm644 ${FLATPAK_ID}.appdata.xml /apps/share/metainfo/${FLATPAK_ID}.metainfo.xml
+    - for icon_size in 512 256 192 128 96 64 48 32 24; do install -Dm644 winbox${icon_size}.png /app/share/icons/hicolor/${icon_size}x${icon_size}/apps/${FLATPAK_ID}.png; done
+    sources:
+    - type: script
+      dest-filename: apply_extra
+      commands:
+        - unzip WinBox_Linux.zip
+        - chmod 755 WinBox
+        - rm -rf assets WinBox_Linux.zip
+    - type: extra-data
+      filename: WinBox_Linux.zip
+      only-arches:
+        - x86_64
+      url: https://download.mikrotik.com/routeros/winbox/4.0beta6/WinBox_Linux.zip
+      sha256: 3c26f58d00f349b95875ff9abcafc8610798803d7d271f3f2b0760f7f120c4db
+      size: 22220141
+    - type: script
+      dest-filename: winbox
+      commands:
+      - exec /app/extra/WinBox $@
+    - type: file
+      path: winbox.desktop
+    - type: file
+      path: com.mikrotik.WinBox.appdata.xml
+    - type: file
+      path: winbox512.png
```

Comment:
only one icon file should be here. (ideal is ZERO everything being upstream). so remove the smaller.